events/templates/events/create_event.html

```diff
@@ -21,12 +20,15 @@
                     <form method="POST">
                         {% csrf_token %}
 
-                        {{ form }}
+                        {{ event_form }}
+                        <br><br>
+
+                        <h3 style="color:white;"> Reminder: </h3>
+                        {{ reminder_form }}
```

Comment:
IMHO i think we should make the reminder form hidden, and only after a user click 'add reminder' button to make it visible.
similiar to what i did on the submit button on the #80,
because by how it looks now, it feels like you have to make a reminder.

Reply:
done. 
created a bell icon that when clicked the form will show itself,
and when clicking again on the bell icon it will hide the form